CI: graphviz failure?

seems to have started failing: https://github.com/ibis-project/ibis/pull/2628/checks?check_run_id=1911463676


```
================================== FAILURES ===================================
_______ TestInteractiveUse.test_repr_png_is_not_none_in_not_interactive _______

self = <ibis.tests.expr.test_interactive.TestInteractiveUse testMethod=test_repr_png_is_not_none_in_not_interactive>

    def test_repr_png_is_not_none_in_not_interactive(self):
        pytest.importorskip('ibis.expr.visualize')
    
        table = self.con.table('functional_alltypes')
    
        with config.option_context(
            'interactive', False
        ), config.option_context('graphviz_repr', True):
>           assert table._repr_png_() is not None
E           AssertionError: assert None is not None
E            +  where None = <bound method Expr._repr_png_ of DatabaseTable[table]\n  name: functional_alltypes\n  schema:\n    id : int32\n    bool_co...\n    date_string_col : string\n    string_col : string\n    timestamp_col : timestamp\n    year : int32\n    month : int32>()
E            +    where <bound method Expr._repr_png_ of DatabaseTable[table]\n  name: functional_alltypes\n  schema:\n    id : int32\n    bool_co...\n    date_string_col : string\n    string_col : string\n    timestamp_col : timestamp\n    year : int32\n    month : int32> = DatabaseTable[table]\n  name: functional_alltypes\n  schema:\n    id : int32\n    bool_col : boolean\n    tinyint_col : int...4\n    date_string_col : string\n    string_col : string\n    timestamp_col : timestamp\n    year : int32\n    month : int32._repr_png_

ibis\tests\expr\test_interactive.py:49: AssertionError
_________________________ test_optional_graphviz_repr _________________________

    @pytest.mark.skipif(
        bool(os.environ.get('APPVEYOR', None)),
        reason='Not sure what the prerequisites for running this on Windows are',
    )
    def test_optional_graphviz_repr():
        t = ibis.table([('a', 'int64'), ('b', 'string'), ('c', 'int32')])
        expr = (
            t.groupby(t.b).aggregate(sum_a=t.a.sum().cast('double')).sort_by('c')
        )
    
        # default behavior
>       assert expr._repr_png_() is not None
E       AssertionError: assert None is not None
E        +  where None = <bound method Expr._repr_png_ of ref_0\nUnboundTable[table]\n  name: unbound_table_87\n  schema:\n    a : int64\n    b : st...ortKey[array-sort]\n      expr:\n        c = Column[int32*] 'c' from table\n          ref_0\n      ascending:\n        True>()
E        +    where <bound method Expr._repr_png_ of ref_0\nUnboundTable[table]\n  name: unbound_table_87\n  schema:\n    a : int64\n    b : st...ortKey[array-sort]\n      expr:\n        c = Column[int32*] 'c' from table\n          ref_0\n      ascending:\n        True> = ref_0\nUnboundTable[table]\n  name: unbound_table_87\n  schema:\n    a : int64\n    b : string\n    c : int32\n\nAggregation[t...SortKey[array-sort]\n      expr:\n        c = Column[int32*] 'c' from table\n          ref_0\n      ascending:\n        True._repr_png_

ibis\tests\expr\test_visualize.py:114: AssertionError
______________________________ test_html_escape _______________________________

    def test_html_escape():
        # Check that we correctly escape HTML <> characters in the graphviz
        # representation. If an error is thrown, _repr_png_ returns None.
        expr = ibis.table([('<a & b>', ibis.expr.datatypes.Array('string'))])
>       assert expr._repr_png_() is not None
E       assert None is not None
E        +  where None = <bound method Expr._repr_png_ of UnboundTable[table]\n  name: unbound_table_93\n  schema:\n    <a & b> : array<string>>()
E        +    where <bound method Expr._repr_png_ of UnboundTable[table]\n  name: unbound_table_93\n  schema:\n    <a & b> : array<string>> = UnboundTable[table]\n  name: unbound_table_93\n  schema:\n    <a & b> : array<string>._repr_png_

ibis\tests\expr\test_visualize.py:152: AssertionError
============================== warnings summary ===============================
```

--- a/environment.yml
+++ b/environment.yml
@@ -15,7 +15,7 @@ dependencies:
   # TODO This section is probably not very accurate right now (some dependencies should probably be in the backends files)
   - sqlalchemy>=1.3
   - dask>=2.22.0
-  - graphviz>=2.38
+  - graphviz>=2.38, <= 2.42.3
   - openjdk=8
   - pytables>=3.6
   - python-graphviz>=0.14, <=0.15

